cmd/fleetctl/apply_test.go

```diff
@@ -937,6 +937,37 @@ spec:
   query: select 1;
   platforms:
     - darwin
+`
+	manualLabelSpec = `---
+apiVersion: v1
+kind: label
+spec:
+  name: manual_label
+  label_membership_type: manual
+  hosts:
+    - host1
+  platforms:
```

Comment:
I wonder what should happen if a user assigns a manual policy that has a `platform` on a host that is not of such platform...

Reply:
Good question, it looks like the platform is mostly ignored/unnecessary for manual labels anyway... Thoughts @marko-lisica or @noahtalerman ?

Reply:
Yep. Just tested and it's currently ignored:
```
cat labels.yml
---
apiVersion: v1
kind: label
spec:
  name: Foobar FTW
  label_membership_type: manual
  hosts:
    - lucass-macbook-pro.local
  platform: linux
```

```
fleetctl apply -f labels.yml
[+] applied 1 labels
```

So in the near future we could warn or fail the request.

Reply:
@mna Good point, I think it makes sense to ignore `platform` for manual labels. I'll document that.